tools/challenge-helper-scripts/utils.ts

```diff
@@ -31,6 +31,14 @@ const createStepFile = ({
   return challengeId;
 };
 
+const createChallengeFile = (
+  title: string,
+  template: string,
+  path = getProjectPath()
+): void => {
+  fs.writeFileSync(`${path}${title}.md`, template);
```

Comment:
Not a big deal, but there's nothing that needs to wait for this, so it could be async.